src/main/java/liquibase/ext/spanner/CloudSpanner.java

```diff
@@ -83,7 +83,14 @@ public String getDefaultDriver(String url) {
     }
     return null;
   }
-  
+
+  protected String getConnectionSchemaName() {
```

Comment:
This is necessary, as Liquibase would otherwise try to create a `CallableStatement` to call a stored procedure to get the current schema name. The current schema name of a Cloud Spanner connection is always the default schema.